bin/_run-docker

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+export USE_VIRTUAL_HOSTS=true
```

Comment:
What is this used for? I think this should probably be set by whatever runs the container and not here.